backend/server.js

```diff
@@ -169,6 +169,13 @@ app.use((req, res, next) => {
 });
 
 app.get('/search', wrap(async (req, res) => {
+  if (macros.DEV && !await Elastic.isConnected()) {
```

Comment:
Very cool! small point - the way its coded right now, `Elastic.isConnected()` is going to be called every time /search is hit in DEV. Could you add a line of code above this fn like so:

```
let elasticConnectedPromise = Elastic.isConnected()
```
and then in this function do :
```
if (macros.DEV && !await elasticConnectedPromise) {
```
- this way Elastic.isConnected() is only called once?

Reply:
Yeah that's probably the way to go.